docs/source/author/topics.rst

```diff
@@ -41,19 +41,16 @@ diagram.
 
 This is just a broad overview of the topics application.
 
-
-.. _viewing-the-current-state:
-
-Viewing the Current State
+Viewing All Topics Content
 ------------------------------------------------------------------------------
 
-When developing locally, once you have a server and ``gulp`` running (see
-:doc:`../getting_started/basic_usage`) you can go to the url below to get a
+When developing locally, once you have run ``csu start`` (see
```

Comment:
well, what if we put `./` everywhere, and then put a note in the troubleshooting page that if this doesn't work, it means they are in the wrong place?